### PR TITLE
Release/0.7.0

### PR DESCRIPTION
**Pull request by jondewijones**
### What

- Update mongo driver
- Update config
- Add lint job to CI

### Who can review

Anyone

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,3 +38,7 @@ test-component:
 convey:
 	goconvey ./...
 
+.PHONY: fixfmt
+fixfmt:
```
Comment (ChrysmOre) [inline Makefile:42]: Any reason why this target wouldn't be just 'fmt'?

File: api/webendpoints_test.go [hunk 2]
```diff
@@ -116,9 +116,10 @@ func hasRoute(r *mux.Router, path, method string) bool {
 }
 
 func createRequestWithAuth(method, url string, body io.Reader) (*http.Request, error) {
```
Comment (ChrysmOre) [inline api/webendpoints_test.go:118]: Does this function need to return an error if it never will?

File: features/steps/steps.go [hunk 3]
```diff
@@ -1,79 +1,76 @@
 package steps
 
 import (
+	"context"
 	"encoding/json"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/pkg/mongodb"
```
Comment (ChrysmOre) [inline features/steps/steps.go:6]: For package names and import handles It's idiomatic to not use mixedCaps, underscores, dashes etc and to keep as simple as possible.. something like `dpmongo` or `mongodriver` would be more appropriate.. But is `mongodb` even unavailable here?

Comment (s-xton) [reply features/steps/steps.go]: package name has been updated to follow current standard

**Review by s-xton — APPROVED** [review]
Raise card to cover fmt makefile and test code change suggestion

**Pull request merged**